packages/inter-protocol/test/auction/test-auctionContract.js

```diff
@@ -280,6 +272,10 @@ const makeAuctionDriver = async (t, customTerms, params = defaultParams) => {
         subscriptionTracker(t, subscribeEach(subscription)),
       );
     },
+    async getReserveBalance(keyword) {
+      const reserveCF = await E.get(reserveKit).creatorFacet;
+      return E.get(E(reserveCF).getAllocations())[keyword];
+    },
```

Comment:
is async really necessary here? would this work:
```suggestion
    getReserveBalance(keyword) {
      const reserveCF = E.get(reserveKit).creatorFacet;
      return E.get(E(reserveCF).getAllocations())[keyword];
    },
```

Reply:
yep, that's better.